routes/api/users.js

```diff
@@ -120,5 +126,22 @@ router.get('/current', authenticateToken, async (req, res, next) => {
   }
 });
 
+router.patch("/avatars", upload.single('avatar'), async (req, res, next) => {
+  try {
+    const { authorization } = req.headers;
+    if (authorization !== 'Bearer {{token}}') {
+      return res
+        .status(401)
+        .json({ message: 'Not authorized' });
+    }
```

Comment:
To pewnie jest w autoryzacji. Czyli przed upload doaj autoryzacje i tutaj juz nie musisz sprawdzac. 
`router.patch("/avatars", auth, upload.single('avatar'), async (req, res, next) => {`